src/interfaces.ts

```diff
@@ -412,9 +412,20 @@ export interface Web3AuthOptions {
   /**
    * Set this flag to false to generate keys on client side
    * by default keys are generated on using dkg protocol on a distributed network
-   * @defaultValue false if keyType is ed25519, true for secp256k1 keys
+   * Note: This option is not supported for ed25519 key type
+   * @defaultValue `true`
    */
   useDKG?: boolean;
+
+  /**
+   * @defaultValue `false` for secp256k1 and `true` for ed25519
```

Comment:
should we have same default value of true for both keytype ?
as the default value changes depend on the type might be confusing

Reply:
also should we have the naming as useTssDKG? 
so that it match the useDKG that added before
or would it be more confusing if change to useTssDKG?

Reply:
> should we have same default value of true for both keytype ? as the default value changes depend on the type might be confusing

yeah it can be confusing but we want to keep dkg as default for secp.

Reply:
> also should we have the naming as useTssDKG? so that it match the useDKG that added before or would it be more confusing if change to useTssDKG?

or shall i make it useClientGeneratedTssKey, i m ignoring dkg because dkg is not supported for ed25519.